samples/pollion_dash/data_preparation.R

```diff
@@ -0,0 +1,178 @@
+lower_bound_birthyear <- 1900
+
+func_integer("birthYear",upper_bound_birthyear,lower_bound_birthyear)
+
+#city of residence
+levels(data$citySize)
+level_reorder_city <- c("less than 5000","5000 - 19999","20000 - 99999","100000 - 1000000","more than 1000000","not answered")
+level_labels_city <- c("weniger als 5000","5000 - 19999","20000 - 99999","100000 - 1000000","mehr als 1000000","keine Angabe")
+
+func_ordinal("citySize",level_reorder_city,level_labels_city)
+
+##################make sure how to deal with socio demographic dublicates!!!!!!!!!!!!
+
+#organicConsumptionFrequency
+level_reorder_consumption=c("daily","weekly","monthly")
+level_label_consumptionlabels=c("taglich","wochentlich","monatlich")
+
+func_ordinal("organicConsumptionFrequency1",level_reorder_consumption,level_label_consumptionlabels)
+
+
+#Pasta Sort
+level_reorder_pasta_sorte=c(levels(data$pasta_sorte)[1],levels(data$pasta_sorte)[4],
+                            levels(data$pasta_sorte)[5],levels(data$pasta_sorte)[3],
+                            levels(data$pasta_sorte)[2])
+
+level_label_pasta_sorte=c(levels(data$pasta_sorte)[1],levels(data$pasta_sorte)[4],
+                          levels(data$pasta_sorte)[5],levels(data$pasta_sorte)[3],
+                          levels(data$pasta_sorte)[2])
+
+func_categorial("pasta_sorte",level_reorder_pasta_sorte,level_label_pasta_sorte)
+
+#Erwartung erfullt
+data$erwartung_erfullt
+level_reorder_erwartung_erfullt<-c("1 (ganz und gar nicht)","2","3","4","5 (voll und ganz)")
+level_label_erwartung_erfullt<-c("1","2","3","4","5")
+
+func_ordinal("erwartung_erfullt",level_reorder_erwartung_erfullt,level_label_erwartung_erfullt)
+
+#bewertung_geschmack
+level_reorder_bewertung_geschmack<-c("1 (sehr schlecht)","2","3","4","5 (sehr gut)")
+level_label_bewertung_geschmack<-c("1","2","3","4","5")
+
+func_ordinal("bewertung_geschmack",level_reorder_bewertung_geschmack,level_label_bewertung_geschmack)
+data$bewertung_geschmack
+
+#bewertung_konsistenz
+level_reorder_bewertung_konsistenz<-level_reorder_bewertung_geschmack
+level_label_bewertung_konsistenz<-level_label_bewertung_geschmack
+
+func_ordinal("bewertung_konsistenz",level_reorder_bewertung_konsistenz,level_label_bewertung_konsistenz)
+
+#wieder kaufen
+level_reorder_wieder_kaufen<-c("1 (bestimmt nicht wieder kaufen)","2","3","4","5 (bestimmt wieder kaufen)")
+level_label_wieder_kaufen<-c("1","2","3","4","5")
+
+func_ordinal("wieder_kaufen",level_reorder_wieder_kaufen,level_label_wieder_kaufen)
+
+#weiter empfehlen
+level_reorder_weiter_empfehlen<-c("Ja","Nein","Keine Angabe")
+level_label_weiter_empfehlen<-c("Ja","Nein","Keine Angabe")
+
+
+table(data[, 11])
+
+
+pal <- brewer.pal(9, "BuGn")
+pal <- pal[-(1:2)]
+png("wordcloud.png", width=1280,height=800)
+#wordcloud(d$word,d$freq, scale=c(8,.3),min.freq=2,max.words=100, random.order=T, rot.per=.15, colors=pal, vfont=c("sans serif","plain"))
+xkcd.corpus <- Corpus(DataframeSource(data.frame(data[, 11])))
```

Comment:
why xkcd in here? maybe change variable name to less comic and confusing one :-)